### ISSUE TITLE
Online doc issue in Chrome Browser on Win 7

### ISSUE THREAD

**Issue by newbienewbiedo** [orientechnologies/orientdb#5891]
Chrome browser ver. 49.0.2623.108 m
I start at http://orientdb.com/docs/last/index.html and left-pane links for Introduction and sections 1-10 appear.

When I click, in the left pane, 2. Data Modeling, the left pane should expand to show items 2.1, 2.2 etc. (as it does in IE).

But instead of doing that, the Introduction, and section 2, 5, 6, 9 and 10 disappear, leaving only main items 1, 3, 4, 7 and 8.  This persists even after hitting the back button.

Not a big deal; I can read docs on IE.  But I thought you'd like to know!

**Comment by smolinari** [comment]
I'd say the problem is somewhere else. I have the same version of Chrome on Windows7 and everything works perfectly for me (except the links in the methods, which I reported a while ago). Try emptying out your browser's cache for ODB's site. That is currently the only thing I can think of. 

Scott

**Comment by newbienewbiedo** [comment]
Thanks Scott, it worked!  For others' future reference I performed two operations (not sure which specific one worked):

I entered the URL chrome://history/ and selected and cleared all orientdb.com pages 

Next I entered the URL chrome://settings/clearBrowserData and selected only the following items and cleared them from the beginning of time:

Cookies and other site and plugin data
Cached images and files
Autofill form data
Hosted app data
Content licenses

But not

Browsing history (other than orientdb.com pages)
Download history
Passwords

Also of interest:
Google's site suggests that the only site-specific removal option is for browsing history.  See
https://support.google.com/chrome/answer/95589?rd=1

Thanks again!
